guides/coveralls-ci-cd/coveralls_ci_cd.md

```diff
@@ -44,8 +46,7 @@ we should use
 Coveralls.wear_merged!('rails')
 ```
 
-Then create a custom rake task that will run all your test suites then submit coverage
-results to the coveralls website.
+Then create a custom rake task that will run all your test suites then submit coverage results to coveralls.
 
 Create a new file `lib/tasks/ci.rake`
 
```

Comment:
Add that they should add the coverage folder to gitignore

Reply:
fixed